build_tools/pkgci/external_test_suite/gpu_rocm_models.json

```diff
@@ -0,0 +1,20 @@
+{
+    "config_name": "gpu_rocm",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=rocm",
+      "--iree-rocm-target-chip=gfx90a",
+      "--iree-opt-const-eval=false",
+      "--iree-codegen-transform-dialect-library=attention_and_matmul_spec.mlir"
```

Comment:
This might be a good time to test both default flags and some "additional flags" (can choose a better name for that).

If default flags fail compilation, that's good to test with XFAIL.